### ISSUE TITLE
undeclared identifier 'std' on Windows build

### ISSUE THREAD

**Issue by joa-quim** [Keno/Cxx.jl#114]
After #113 I get these warnings

>  WARNING: Could not add directory C:/mingw-builds/x64-4.8.1-win32-seh-rev5/mingw64/x86_64-w64-mingw32/include to clang search path!
>  WARNING: Could not add directory C:/mingw-builds/x64-4.8.1-win32-seh-rev5/mingw64/lib/gcc/x86_64-w64-mingw32/4.8.1/include/c++ to clang search path!
>  WARNING: Could not add directory C:/mingw-builds/x64-4.8.1-win32-seh-rev5/mingw64/lib/gcc/x86_64-w64-mingw32/4.8.1/include/c++/x86_64-w64-mingw32 to clang search path!

It has to fail because I don't have any C:/mingw-builds/x64-4.8.1-win32-seh-rev5/...

Don't know if it's related to the warnings but I now get an error on std not being known

>   julia> cxx"""
>   void printme(int x) {
>   std::cout << x << std::endl;
>   }
>  """
>   __cxxjl_1.cpp:2:8: error: use of undeclared identifier 'std'
>   std::cout << x << std::endl;

**Comment by Keno** [comment]
The problem is that it can't find the C++ standard library headers. If you look in the code there's a bunch of places where it tries to find the standard headers on the various operating systems. That needs a new case for windows.

**Comment by joa-quim** [comment]
Right. I think you mean the code in ``initialization.jl``. Changed it to reflect my path and now the next issue is:

> julia> using Cxx
> Warning: error initializing module Cxx:
> BoundsError(a=Array{Cxx.ClangCompiler, 1}[], i=(1,))
